### PR TITLE
create _tkinter stub

### PR DESCRIPTION
**Pull request by Akuli**
checks a checkbox in #1019

I thought I'd do some smaller tkinter PRs because #4363 doesn't seem to be getting merged in a while (needs conversation)

### CODE REVIEW

**Comment by Akuli** [comment]
How do I tell stubtest that an attribute comes from a `__getattr__`? By overloading `__getattr__` with Literals?

**Comment by hauntsaninja** [comment]
stubtest is okay with attributes generally, because those are often defined on instances, so it can't get at them without instantiating. It does assume methods are not dynamically created though.
You have two options:
a) you can annotate your dynamic methods as attributes, eg: `eval: Callable[[str], Any]` (or for more complicated things, of type callback protocol). Depending on what __getattr__ is doing, this might actually be more honest.
b) just add it to the stubtest whitelists (which I think is fine doing in this case). for an example in a similar vein, we whitelisted all of ast.NodeVisitor's visit_* methods.

Really appreciate all your improvements!

**Comment by Akuli** [comment]
ready for review

**Comment by Akuli** [comment]
How about putting the methods to a class named `_Something` and inheriting the two classes from that? This way there would be no copy/pasta.

File: stdlib/3/_tkinter.pyi [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+    setvar: Any
+    split: Any
+    splitlist: Any
+    unsetvar: Any
+    wantobjects: Any
+    willdispatch: Any
+
+# These come from dir(_tkinter). I don't know when actually using any of this
+# would be convenient
+ALL_EVENTS: int = ...
```
Comment (srittau) [inline stdlib/3/_tkinter.pyi:104]: Nit (also applies to assignments below): `= ...` is redundant and should be left out.

**Comment by Akuli** [comment]
I made the comments more concise.

I can see how to apply these commenting guidelines to some of my comments in #4363, for example, and I will change that PR appropriately next. But sometimes I really want people working on the stubs to see something, even if they're adding something new rather than modifying something that I wrote. Can I still use comments for that?

**Pull request merged**